runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentView.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2013 The Chromium Authors. All rights reserved.
+// Copyright (c) 2013-2014 Intel Corporation. All rights reserved.
```

Comment:
2015 is enough for new created file.

For the chromium copyright, I assume you copied the original code from Chromium, correct? If not, let's remove that copyright line.